code/modules/antagonists/abductor/machinery/camera.dm

```diff
@@ -76,6 +76,11 @@
 	var/mob/living/carbon/human/C = owner
 	var/mob/camera/ai_eye/remote/remote_eye = C.remote_control
 	var/obj/machinery/abductor/pad/P = target
+
+	if(istype(get_area(remote_eye), /area/ai_monitored/turret_protected/ai))
```

Comment:
what? Don't we have `noteleport` flags for areas or something that does exactly this? Or does the abductor console bypass that?

Reply:
It bypasses, I swear I read something along those lines back in the day but I searched for protected and couldn't find it.

Reply:
Noteleport also prevents teleportation though